.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,89 @@
+name: build-docs
+on: push
+
+env:
+  DEPLOY_BRANCH: gh-pages               # deployed docs branch
+
+defaults:
+  run:
+    shell: bash -le {0}
+jobs:
+  # hello-world-job:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - name: echo-hello-world
+  #       run: |
+  #         echo "Hello World!"
+  #         uname -a
+  #         lsb_release -a
+
+  build-sphinx-html:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+        
+      - name: Setup Mambaforge
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          miniforge-version: latest
+          activate-environment: stardis
+          use-mamba: true
+      
+      - name: Cache Environment Lockfile
+        id: cache-env-lockfile
+        uses: actions/cache@v3
+        with:
+          path: /usr/share/miniconda3/envs/stardis
+          key: conda-linux-64-lockfile
+                  
+      - name: Update Conda Environment
```

Comment:
Maybe this is standard and I'm not aware, but why do we build the environment from a cached lockfile and then update it from tardis releases, rather than just building it from downloaded file in the first place?